README.md

```diff
@@ -49,8 +49,8 @@ over the multi-asset order book to obtain a list of matches. The matches are the
 translated into transactions that will be signed and submitted by the bot.
 
 Due to the open and decentralized design of the protocol, anybody can run a Smart Order
-Router instance and collect a share of the fees, thus running a Smart Order Router instance
-is not only contributiung to the further decentralization of the protocol, but it is also
+Router instance and collect the arbitrage opportunities, thus running a Smart Order Router instance
```

Comment:
> collect the arbitrage opportunities,
I think we should change the wording here. One does not "collect" opportunities.

Reply:
Sorry, could you elaborate / suggest the required changes?